cmd/podman/images/scp.go

```diff
@@ -343,16 +343,25 @@ func execPodman(podman string, command []string) error {
 		logrus.Debug("Executing podman command")
 		return cmd.Run()
 	}
-	machinectl, err := exec.LookPath("machinectl")
+	cmdLogin, err := utils.LoginUser("root")
+	if err != nil {
+		return err
+	}
+	sd, err := exec.LookPath("systemd-run")
+	fmt.Println("HERE WE ARE")
 	if err != nil {
 		cmd := exec.Command("su", "-l", "root", "--command")
 		cmd = utils.CreateSCPCommand(cmd, []string{strings.Join(command, " ")})
 		return cmd.Run()
 	}
-	cmd := exec.Command(machinectl, "shell", "-q", "root@.host")
+	cmd := exec.Command(sd, "--pty", "-q", "--wait", "--uid", "0")
 	cmd = utils.CreateSCPCommand(cmd, command)
-	logrus.Debug("Executing load command machinectl")
-	return cmd.Run()
+	logrus.Debug("Executing load command systemd")
```

Comment:
Hint: if something goes wrong here, the maintenance programmer might really appreciate seeing the actual command string that was invoked.